content/applications/inventory_and_mrp/inventory/routes/strategies/putaway.rst

```diff
@@ -1,54 +1,208 @@
-=======================
-What is a Putaway Rule?
-=======================
+=============
+Putaway Rules
+=============
 
-A good warehouse implementation takes care that products automatically move to their appropriate
-destination location. To make that process easier, Odoo uses *Putaway rules*. But what is a putaway
-rule? Putaway is the process of taking products off the receiving shipments and directly putting
-them into the most appropriate location.
+Putaway is the process of routing products to appropriate storage locations upon shipment arrival.
```

Comment:
Consider italicizing this since you're defining the word "putaway"
```suggestion
*Putaway* is the process of routing products to appropriate storage locations upon shipment arrival.
```